app/views/spam2/_insights.html.erb

```diff
@@ -63,153 +63,153 @@
 <script src="https://cdn.jsdelivr.net/npm/apexcharts"></script>
 <script>
 var options = {
-	chart: {
-		type: "line",
-		height: 300,
-		foreColor: "#999",
-		scroller: {
-			enabled: true,
-			track: {
-				height: 7,
-				background: '#e0e0e0'
-			},
-			thumb: {
-				height: 10,
-				background: '#94E3FF'
-			},
-			scrollButtons: {
-				enabled: true,
-				size: 9,
-				borderWidth: 2,
-				borderColor: '#008FFB',
-				fillColor: '#008FFB'
-			},
-			padding: {
-				left: 30,
-				right: 20
-			}
-		}
-	},
-	colors: ['#E8D020', '#E85C20', '#165F8B'],
-	stroke: {
-		curve: "smooth",
-		width: 3
-	},
-	dataLabels: {
-		enabled: false
-	},
-	series: [{
-		name: 'Flagged Nodes',
-		data: generateWeekWiseTimeSeries(0)
-	}, {
-		name: 'Spammed Nodes',
-		data: generateWeekWiseTimeSeries(1)
-	}, {
-		name: 'Unmoderated  Nodes',
-		data: generateWeekWiseTimeSeries(2)
-	}],
-	markers: {
-		size: 0,
-		strokeColor: "#fff",
-		strokeWidth: 3,
-		strokeOpacity: 1,
-		fillOpacity: 1,
-		hover: {
-			size: 6
-		}
-	},
-	xaxis: {
-		type: "datetime",
-		axisBorder: {
-			show: false
-		},
-		axisTicks: {
-			show: false
-		}
-	},
-	yaxis: {
-		tickAmount: 6,
-		min: 0,
-		opposite: true,
-		labels: {
-			offsetX: 24,
-			offsetY: -5
-		},
-		tooltip: {
-			enabled: true
-		}
-	},
-	grid: {
-		padding: {
-			left: -5,
-			right: 5
-		}
+  chart: {
+    type: "line",
+	height: 300,
+	foreColor: "#999",
+	scroller: {
+	  enabled: true,
+	  track: {
+	    height: 7,
+	    background: '#e0e0e0'
+	  },
+	  thumb: {
+	    height: 10,
+	    background: '#94E3FF'
+	  },
+	  scrollButtons: {
+	    enabled: true,
+	    size: 9,
+	    borderWidth: 2,
+	    borderColor: '#008FFB',
+	    fillColor: '#008FFB'
+	  },
+	  padding: {
+		left: 30,
+		right: 20
+	  }
+	}
+  },
+  colors: ['#E8D020', '#E85C20', '#165F8B'],
+  stroke: {
+    curve: "smooth",
+    width: 3
+  },
+  dataLabels: {
+    enabled: false
+  },
+  series: [{
+    name: 'Flagged Nodes',
+    data: generateWeekWiseTimeSeries(0)
+  }, {
+	name: 'Spammed Nodes',
+	data: generateWeekWiseTimeSeries(1)
+  }, {
+	name: 'Unmoderated  Nodes',
+	data: generateWeekWiseTimeSeries(2)
+  }],
+  markers: {
+    size: 0,
+    strokeColor: "#fff",
+    strokeWidth: 3,
+    strokeOpacity: 1,
+    fillOpacity: 1,
+    hover: {
+	  size: 6
```

Comment:
This too.

Reply:
I am surprised it appeared like this here, It's totally different from what I pushed from my Local Machine. I would have to do the editing via Github so as to avoid this error. I will make the corrections and get back to you @TildaDares